docs/user/administration/users/README.md

```diff
@@ -0,0 +1,9 @@
+# BuddyPress Administration - Users
+
+BuddyPress adds various sub-menus to the WordPress Users' one to help you manage community members related features.
+
+- Advanced profile
```

Comment:
In order to maintain consistency with menu items, tabs, etc., of which should hold precedence, unless there is a good reason to change precedence.
```suggestion
- Extended Profile
```